mod_reforged/msu_systems/mod_settings.nut

```diff
@@ -15,4 +15,5 @@ tacticalTooltipPage.addEnumSetting("TacticalTooltip_BagItems", "All", ["All", "A
 
 tacticalTooltipPage.addRangeSetting("CollapseEffectsWhenX", 20, 0, 20, 1, "Collapse Effects When", "While the number of effects is below this value all effects display their icon and use a separate line. Otherwise they combine into a single block of text in order to save space.");
 tacticalTooltipPage.addRangeSetting("CollapsePerksWhenX", 7, 0, 20, 1, "Collapse Perks When", "While the number of perks is below this value all perks display their icon and use a separate line. Otherwise they combine into a single block of text in order to save space.");
+tacticalTooltipPage.addBooleanSetting("DisplayPerksTwice", true, "Display Perks Twice", "All perks will always be listed under 'Perks', even if they also appear under 'Effects'. Turning this option off will instead hide duplicate entries from the category 'Perks'.");
```

Comment:
The name **"Show Status Perk and Effect"** doesn't seem too clear to me.
It can be quickly confused with the **"Show Effects"** and **"Show Perks"** setting.